istioctl/cmd/proxyconfig.go

```diff
@@ -61,6 +158,22 @@ func setupConfigdumpEnvoyConfigWriter(podName, podNamespace string, out io.Write
 	return cw, nil
 }
 
+func setupLoggingEnvoyConfigWriter(param, podName, podNamespace string) ([]byte, error) {
```

Comment:
Naming isn't quite right here. This changes the envoy logging levels. It doesn't setup a logging config writer. Maybe `setEnvoyLoggingConfig`?